igneous/tasks/skeleton.py

```diff
@@ -223,6 +240,9 @@ def decompress_all_labels():
     for sid, skel in skeletons.items():
       skel.id = sid
 
+    if self.split_at_branches:
```

Comment:
moving this after cross section might give slightly nicer cross section results near the branch points but probably very minor